### PR TITLE
Tag Clang v0.2.0 and v0.3.0

### PR DESCRIPTION
**Pull request by ihnorton**

### CODE REVIEW

File: Clang/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.6-
```
Comment (tkelman) [inline Clang/versions/0.2.0/requires]: shouldn't claim to support early dev versions where `abstract type` wouldn't work

Comment (ihnorton) [inline Clang/versions/0.2.0/requires]: fixed. thanks.

File: Clang/versions/0.1.1/requires [hunk 2]
```diff
@@ -0,0 +1,2 @@
+julia 0.5- 0.6
```
Comment (tkelman) [inline Clang/versions/0.1.1/requires]: none of the earlier versions have upper bounds, is this needed if 0.2.0 is available?

**Comment by tkelman** [comment]
It's unlikely that you'll ever make another 0.4 release, but this should probably be 0.2.0 and 0.3.0 rather than 0.1.1 and 0.2.0

**Pull request merged**